cmds/core/uptime/uptime_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"testing"
+	"time"
+)
+
+type UptimeOutput struct {
```

Comment:
You don't actually have to make these named structs, but this isn't too much of a problem. Once you move to table driven tests, you'll see that it's not that necessary

Reply:
Should I change the way this from struct or its fine to do it this way?

Reply:
It's fine to leave it, you can change it when you move to table driven tests.

Reply:
Done